app/controllers/thredded/autocomplete_users_controller.rb

```diff
@@ -6,17 +6,27 @@ class AutocompleteUsersController < Thredded::ApplicationController
     def index
       authorize_creating PrivateTopicForm.new(user: thredded_current_user).private_topic
       users = params.key?(:q) ? users_by_prefix : users_by_ids
+
       render json: {
-        results: users.map do |user|
-          { id:         user.id,
-            name:       user.send(Thredded.user_name_column),
-            avatar_url: Thredded.avatar_url.call(user) }
-        end
+          results: users.map do |user|
+            { id:         user.id,
+              name:       display_name(user),
+              avatar_url: Thredded.avatar_url.call(user) }
+          end
       }
     end
 
     private
 
+    # Defines how the name is displayed in the autocomplete search results
+    def display_name(user)
+      if Thredded.user_display_name_method == :to_s
+        user.send(Thredded.user_name_column)
+      else
+        "#{user.send(Thredded.user_name_column)} - #{user.send(Thredded.user_display_name_method)}"
+      end
+    end
+
```

Comment:
So what I did for now is just fork off into another branch to update the [controller](https://github.com/jvoss/thredded/blob/bluesky/app/controllers/thredded/autocomplete_users_controller.rb) for my project. I wound up creating a method in the controller (#autocomplete_display_name) that provides the original functionality I thought you guys might like. 

I'm sure there is further tweaking (as in a configuration option) but basically as it stands, if the user is using anything other than #to_s for the display name method it will include their custom display method in addition to their forum user name.

Take a look at that and let me know what you guys think. If you have suggestions on how you might like it included (if at all) I'm certainly open to suggestions.